opensrp-app/src/main/java/org/smartregister/repository/EventClientRepository.java

```diff
@@ -255,11 +257,45 @@ public Boolean checkIfExists(Table table, String baseEntityId, SQLiteDatabase sq
         return false;
     }
 
+    public void populateFormSubmissionIds() {
```

Comment:
This should check for submissionids from those passed in the array. Also return the set in this method

Reply:
What if there are events already saved?